webapp/src/components/sidebar_right/index.tsx

```diff
@@ -127,6 +127,7 @@ function SidebarRight({theme}: {theme: Theme}) {
         };
     });
 
+    // States used for storing the updated details of PRs, reviews, and RHSState after being taken from the redux store.
```

Comment:
This comment fails to explain the purpose of these variables so modify the comment according to the suggestions I gave in other comments.